src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/page/RichTextEditorTextArea.kt

```diff
@@ -0,0 +1,59 @@
+package com.atlassian.performance.tools.jiraactions.api.page
+
+import com.atlassian.performance.tools.jiraactions.Patience
+import org.apache.logging.log4j.LogManager
+import org.apache.logging.log4j.Logger
+import org.openqa.selenium.By
+import org.openqa.selenium.JavascriptExecutor
+import org.openqa.selenium.WebDriver
+import org.openqa.selenium.WebElement
+import org.openqa.selenium.support.ui.ExpectedConditions
+import org.openqa.selenium.support.ui.ExpectedConditions.*
+import java.time.Duration
+
+/**
+ * Helper class for Rich Text Editor enabled Textarea
+ *
+ */
+class RichTextEditorTextArea(
+    private val driver: WebDriver
+) {
+
+    private val logger: Logger = LogManager.getLogger(this::class.java)
+
+    private fun clearAndTypeIfPresent(id :String, text: String) {
```

Comment:
So this named consistently with `EditIssuePage` internals, which by itself is good.
However, I think we set up a bad example there. According to [SRP](https://en.wikipedia.org/wiki/Single_responsibility_principle), methods should be responsible for one thing only. Any time you see `and`, you should become suspicious - it has multiple responsibilities: 1. clearing, 2. typing.

Naturally we still want to do these two operations underneath, but let's look for a consistent, higher-level description of that. Why are we doing these two things? Because we want to *overwrite* the text.
So I'd rename this to `overwriteIfPresent`.